apps/web/app/vagas/page.tsx

```diff
@@ -27,53 +27,48 @@ export default function Page() {
 
   const jobList = localOpenings.concat(globalOpenings) as JobArticleProps[]
 
-  const availableLocations = [
-    ...new Set(jobList.map((job) => job.location).flat()),
-  ]
-  const availableLanguages = [
-    ...new Set(jobList.map((job) => job.language).flat()),
-  ]
-  const availableSkills = [...new Set(jobList.map((job) => job.skills).flat())]
-
-  const filteredList = jobList.filter((job) => {
-    const locations = availableLocations.filter((location) => {
-      if (selectedLocations.length) {
-        return selectedLocations.includes(location)
-      }
-
-      return true
-    })
-
-    const languages = availableLanguages.filter((language) => {
-      if (selectedLanguages.length) {
-        return selectedLanguages.includes(language)
-      }
-
-      return true
-    })
-
-    const skills = availableSkills.filter((skill) => {
-      if (selectedSkills.length) {
-        return selectedSkills.includes(skill)
-      }
-
-      return true
-    })
-
-    const searchByTitle = searchText
-      ? job.title.toLowerCase().includes(searchText.toLowerCase())
-      : true
-
-    return (
-      searchByTitle &&
-      locations.includes(job.location) &&
-      languages.includes(job.language) &&
-      job.skills
-        .map((skill) => skills.includes(skill))
-        .filter((item) => item)[0]
-    )
+  const available = {
+    locations: new Set<string>(),
+    languages: new Set<string>(),
+    skills: new Set<string>(),
+  }
+
+  jobList.map((job) => {
```

Comment:
I just change the `map` for `forEach` function if you will not use the array which `map` returns

Reply:
done